pkg/issuer/acme/http/istio.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2020 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package http
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	cmacme "github.com/jetstack/cert-manager/pkg/apis/acme/v1"
+	logf "github.com/jetstack/cert-manager/pkg/logs"
+	istioapinetworking "istio.io/api/networking/v1beta1"
+	istioclientnetworking "istio.io/client-go/pkg/apis/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func (s *Solver) ensureIstio(ctx context.Context, ch *cmacme.Challenge, svcName string) (bool *istioclientnetworking.VirtualService, err error) {
```

Comment:
Having a variable name of `bool` is quite confusing :sweat_smile: 

I think we should remove the variable names in the returns all together:

```go
(*istioclientnetworking.VirtualService, error)
```